app/views/questions/show.html.erb

```diff
@@ -1,17 +1,18 @@
 <% content_for :title, @question.title.truncate(50) %>
 
 <%= render 'posts/expanded', post: @question %>
-<span class="clearfix"></span><br/>
-<h4><%= pluralize(@question.answers.where(deleted: false).count, "answer") %></h4>
 
-<%= tabs do %>
-  <% tab 'Score', query_url(sort: 'score'), is_active: active_search?('score') %>
-  <% tab 'Age', query_url(sort: 'age'), is_active: active_search?('created_at') %>
-<% end %>
+<h2><%= pluralize(@question.answers.where(deleted: false).count, "answer") %></h2>
+
+<div class="button-list is-gutterless has-float-right">
+  <a href="<%= query_url(sort: 'score') %>" class="button is-muted is-outlined <%= active_search?('score') ? 'is-active' : '' %>">Score</a>
+  <a href="<%= query_url(sort: 'age') %>" class="button is-muted is-outlined <%= active_search?('created_at') ? 'is-active' : '' %>">Active</a>
+</div>
+
+<div class="has-clear-clear"></div>
```

Comment:
This should probably be feedback on Co-Design rather than here, but still: component styles should have the clear built into them (or better, should use flexbox or grid so that they don't need it in the first place). It shouldn't be up to the calling view to add the clears in the right places.

Reply:
The clear is neccessary, because I floated the element a few lines above ("has-float-right"). That seems correct to me: When I add a float via a utility, I should need one to remove it again. Also, not all button lists need to be floated.

Reply:
Ah, gotcha. Yeah, that makes sense. Can we make the `has-clear-clear` class work on the same element so that we don't have to add another element? Small thing, but I believe there's a way to do it.